### DISCUSSION TITLE
Does the Script still have more resources to receive or has it already reached its peak?

### DISCUSSION THREAD

**Discussion by ricardo404** [redphx/better-xcloud#27] [General]
Hey @redphx , is it still possible to have more resources in the Script or has it already reached a level where it encounters the limitations of Xcloud?

**Comment by ricardo404** [comment]
https://www.reddit.com/r/xcloud/comments/15al3uv/op%C3%A7%C3%A3o_oculta_do_edge_para_melhroar_o_input_lag_o/

Does it work? I tested it here, but I have to look closely, running a serious risk of becoming a placebo effect (head thing).
According to the tutorial:
Enable "Disable Cache" and use Safari-Mac, as Useragent...
@redphx

**Comment by redphx** [comment]
I think it's just a placebo effect. He didn't provide any stats to back his claim. You could follow his guide and turn on the stats bar in Better xCloud to see if it changes anything.  

Changing User-Agent to Safari will disable some features in xCloud (like voice chat), and it might improve performance a little (I'm just guessing here).

**Comment by ricardo404** [comment]
Hi Red, I tested it but I didn't notice anything noticeable. It could be more of a placebo effect, where we try to improve something by our fragile imagination.
